api/soap/mantisconnect.wsdl

```diff
@@ -163,6 +163,7 @@
    <xsd:element name="monitors" type="tns:AccountDataArray" minOccurs="0"/>
    <xsd:element name="sticky" type="xsd:boolean" minOccurs="0"/>
    <xsd:element name="tags" type="tns:ObjectRefArray" minOccurs="0"/>
+   <xsd:element name="token" type="xsd:string" minOccurs="0"/>
```

Comment:
Are you expecting just to return this when getting issues and ignore it when add/updating issues?  I don't know if "token" is a meaningful name.  Should it be just the random token or the full url? e.g. "share_url" or "anonymous_share_url", etc.